hare/eligibility/oracle.go

```diff
@@ -516,7 +516,6 @@ func (o *Oracle) IsIdentityActiveOnConsensusView(ctx context.Context, edID types
 	}()
 	actives, err := o.actives(ctx, layer)
 	if err != nil {
-		o.WithContext(ctx).With().Error("error getting active set", layer, log.Err(err))
```

Comment:
Why is this log removed?

Reply:
because the returned error is already logged at call sites.